src/docfx/lib/markdown/MarkdownUtility.cs

```diff
@@ -136,6 +148,11 @@ private static string GetToken(string key)
             return t_status.Value.Peek().Context.TemplateEngine.GetToken(key);
         }
 
+        private static void LogInfo(string code, string message, MarkdownObject origin, int? line)
+        {
+            ErrorLog.ConsoleLog(ErrorLevel.Off, new Error(ErrorLevel.Off, code, message, origin.ToSourceInfo(line)));
```

Comment:
oops, maybe this is enough
```suggestion
            Log.Write($"{code} {message}");
```

Reply:
yep